ddht/v5_1/client.py

```diff
@@ -244,6 +245,7 @@ async def send_found_nodes(
             enrs, max_payload_size=FOUND_NODES_MAX_PAYLOAD_SIZE
         )
         num_batches = len(enr_batches)
+        assert enr_batches
```

Comment:
[antipattern](https://github.com/ethereum/snake-charmers-tactical-manual/blob/4dc11553939675e427e5daec51665033b0777538/antipatterns.md#assertions-in-runtime-code)? Is there a special case to use a bare assert here?

Reply:
debugging leftover, thanks for catching it.